src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -180,7 +183,7 @@ private boolean useMetric(MetricRule rule, Metric metric) {
         String dimensionValue = dimension.getValue();
         if (dimensionSelectKeys.contains(dimensionName)) {
           List<String> allowedDimensionValues = rule.awsDimensionSelect.get(dimensionName);
-          if (!allowedDimensionValues.contains(dimensionValue)) {
```

Comment:
http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/billing-metricscollected.html See the currency dimension.